world/stat_checks/check_maker.py

```diff
@@ -388,6 +388,123 @@ def _get_fail_rolls(self):
         else:
             return [obj for obj in rolls if obj.value < 0]
 
+    def get_context(self) -> dict:
+        crit = None
+        botch = None
+        if self.natural_roll_type:
+            if self.natural_roll_type.is_crit:
+                crit = self.natural_roll_type
+            else:
+                botch = self.natural_roll_type
+
+        return {
+            "character": self.character,
+            "npc_name": self.npc_name,
```

Comment:
changing "character" from `self.character` to `self.npc_name` removes the need to change the template

Reply:
I didn't think of that.  I'll have to look at that tomorrow when I'm more lucid.